gui/cloudconnectionwizard.cpp

```diff
@@ -0,0 +1,669 @@
+		showStepQuickMode1();
+		break;
+
+	case Step::QUICK_MODE_STEP_2:
+		showStepQuickMode2();
+		break;
+
+	case Step::QUICK_MODE_SUCCESS:
+		showStepQuickModeSuccess();
+		break;
+
+	case Step::MANUAL_MODE_STEP_1:
+		showStepManualMode1();
+		break;
+
+	case Step::MANUAL_MODE_STEP_2:
+		showStepManualMode2();
+		break;
+
+	case Step::MANUAL_MODE_FAILURE:
+		showStepManualModeFailure();
+		break;
+
+	case Step::MANUAL_MODE_SUCCESS:
+		showStepManualModeSuccess();
+		break;
+	}
+
+	reflowLayout();
+	g_gui.scheduleTopDialogRedraw();
+}
+
+// mode select
+
+void CloudConnectionWizard::showStepModeSelect() {
+	_headlineLabel->setLabel(_("Cloud Connection Wizard"));
+	showContainer("ConnectionWizard_ModeSelect");
+
+	_quickModeButton = new ButtonWidget(_container, "ConnectionWizard_ModeSelect.QuickModeButton", _("Quick mode"), Common::U32String(), kCloudConnectionWizardQuickModeButtonCmd);
+	_manualModeButton = new ButtonWidget(_container, "ConnectionWizard_ModeSelect.ManualModeButton", _("Manual mode"), Common::U32String(), kCloudConnectionWizardManualModeButtonCmd);
+
+#ifdef USE_SDL_NET
+	_quickModeLabel = new StaticTextWidget(_container, "ConnectionWizard_ModeSelect.QuickModeHint", _("Will ask you to run Local Webserver"));
+#else
+	_quickModeLabel = new StaticTextWidget(_container, "ConnectionWizard_ModeSelect.QuickModeHint", _("Requires Local Webserver feature"), Common::U32String(), ThemeEngine::kFontStyleNormal);
+	_quickModeLabel->setEnabled(false);
+	_quickModeButton->setEnabled(false);
+#endif // USE_SDL_NET
+}
+
+void CloudConnectionWizard::hideStepModeSelect() {
+	hideContainer();
+	removeWidgetChecked(_quickModeButton);
+	removeWidgetChecked(_manualModeButton);
+	removeWidgetChecked(_quickModeLabel);
+}
+
+// quick mode
+
+void CloudConnectionWizard::showStepQuickMode1() {
+	_headlineLabel->setLabel(_("Quick Mode: Step 1"));
+	showContainer("ConnectionWizard_QuickModeStep1");
+	showBackButton();
+	showNextButton();
+
+	_label0 = new StaticTextWidget(_container, "ConnectionWizard_QuickModeStep1.Line1", _("In this mode, Local Webserver must be running,"));
+	_label1 = new StaticTextWidget(_container, "ConnectionWizard_QuickModeStep1.Line2", _("so your browser could forward data to ScummVM"));
+
+	_button0 = new ButtonWidget(_container, "ConnectionWizard_QuickModeStep1.RunServerButton", Common::U32String(), Common::U32String(), kCloudConnectionWizardRunServerButtonCmd);
+	_label2 = new StaticTextWidget(_container, "ConnectionWizard_QuickModeStep1.ServerInfoLabel", Common::U32String());
+
+	refreshStepQuickMode1();
+}
+
+void CloudConnectionWizard::refreshStepQuickMode1(bool displayAsStopped) {
+	bool serverIsRunning = false;
+#ifdef USE_SDL_NET
+	serverIsRunning = LocalServer.isRunning();
+#endif // USE_SDL_NET
+	if (displayAsStopped)
+		serverIsRunning = false;
+
+	if (_nextStepButton)
+		_nextStepButton->setEnabled(serverIsRunning);
+
+	if (_button0) {
+		_button0->setLabel(_(serverIsRunning ? "Stop server" : "Run server"));
```

Comment:
I think these should be marked for translation. e.g. wrap them into `_()`. Same for the buttons below.